src/components/country-select.vue

```diff
@@ -1,55 +1,55 @@
 <script>
-import Vue from 'vue'
-import regions from 'country-region-data'
+  import Vue from 'vue'
+  import regions from 'country-region-data'
 
-export default Vue.extend({
-  name: 'CountrySelect',
-  props: ['country', 'topCountry', 'countryName', 'whiteList', 'blackList'],
-  computed: {
-    countries() {
-      let countryList = regions.filter((region) => {
-        if (this.countryName) {
-          return region.countryName !== this.topCountry
-        } else {
-          return region.countryShortCode !== this.topCountry
-        }
-      })
-      if (this.whiteList) {
-        countryList = countryList.filter((country) => {
-          return this.whiteList.includes(country.countryShortCode)
-        })
-      }
-      if (this.blackList) {
-        countryList = countryList.filter((country) => {
-          return !this.blackList.includes(country.countryShortCode)
+  export default Vue.extend({
+    name: 'CountrySelect',
+    props: ['country', 'topCountry', 'countryName', 'whiteList', 'blackList', 'class'],
+    computed: {
+      countries() {
+        let countryList = regions.filter((region) => {
+          if (this.countryName) {
+            return region.countryName !== this.topCountry
+          } else {
+            return region.countryShortCode !== this.topCountry
+          }
         })
+        if (this.whiteList) {
+          countryList = countryList.filter((country) => {
+            return this.whiteList.includes(country.countryShortCode)
+          })
+        }
+        if (this.blackList) {
+          countryList = countryList.filter((country) => {
+            return !this.blackList.includes(country.countryShortCode)
+          })
+        }
+        return countryList
+      },
+      valueType() {
+        return this.countryName ? 'countryName' : 'countryShortCode'
       }
-      return countryList
-    },
-    valueType() {
-      return this.countryName ? 'countryName' : 'countryShortCode'
-    }
-  },
-  methods: {
-    onChange(country) {
-      this.$emit('input', country)
     },
-    topCountryName() {
-      const regionObj = regions.find((region) => {
+    methods: {
+      onChange(country) {
+        this.$emit('input', country)
+      },
+      topCountryName() {
+        const regionObj = regions.find((region) => {
           if (this.countryName) {
             return region.countryName === this.topCountry
           } else {
             return region.countryShortCode === this.topCountry
           }
-      })
-      return regionObj.countryName
+        })
+        return regionObj.countryName
+      }
     }
-  }
-})
+  })
 </script>
 
 <template>
-  <select @change="onChange($event.target.value)">
+  <select @change="onChange($event.target.value)" class="class">
```

Comment:
please change class to :class="className". You need the : to achieve what you are going for here.